src/main/java/com/jalasoft/webservice/model/OCRExtractor.java

```diff
@@ -1,44 +1,48 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
 package com.jalasoft.webservice.model;
 
 import java.io.File;
-
-
+import java.io.IOException;
 import net.sourceforge.tess4j.ITesseract;
 import net.sourceforge.tess4j.TesseractException;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
-
 import net.sourceforge.tess4j.Tesseract;
 
 /**
- * Copyright (c) 2019 Jalasoft.
+ * The class implements methods to convert PDF files into image files, Creates the Controller Model using Tesseract
+ * with the wrapper Tess4J.
  *
- * This software is the confidential and proprietary information of Jalasoft.
- * ("Confidential Information"). You shall not
- * disclose such Confidential Information and shall use it only in
- * accordance with the terms of the license agreement you entered into
- * with Jalasoft.
+ * @author Andy Bazualdo on 9/23/19.
+ * @version 1.0
  */
-public class OCRExtractor {
+public class OCRExtractor implements IConvert {
+    private OCRCriteria ocrCriteria;
+
     /**
-     * Creates the Controller Model using Tesseract with the wrapper Tess4J.
      *
```

Comment:
Your copyrigth in your method needs to explain what action is the is performing.